packages/docs/src/scss/components/_tile.scss

```diff
@@ -13,7 +12,7 @@
 .c-tile__body {
 	padding: 2rem;
 	position: relative;
-	z-index: 1;
+	z-index: 1; // TODO: Evaluate whether this declaration is (still) necessary
```

Comment:
Do we want to add a TODO to our codebase?

Reply:
I‘m not against it in general, but just haven‘t been able to solve this puzzle for now. That for it seemed reasonable for me, it‘s mainly like a marker to a point in the code that either I, somebody else or the previous codeowner would solve.